core/src/debug_client.rs

```diff
@@ -0,0 +1,71 @@
+//! This module contains code for an http client that is used for the VSCode debug plugin.
+
+use prost::Message;
+use reqwest;
+use std::time::Duration;
+use temporal_sdk_core_protos::temporal::api::history::v1::History;
+
+const CLIENT_NAME: &str = "temporal-core";
+const CLIENT_VERSION: &str = "0.1.0";
+
+/// Struct representing the internal debug client
```

Comment:
This sort of comment isn't telling you much. Prefer instead something like this
```suggestion
/// A client for interacting with the VSCode debug plugin
```